### DISCUSSION TITLE
iPXE support for newer, non-CBC ciphers?

### DISCUSSION THREAD

**Discussion by VecnaLarry** [ipxe/ipxe#666] [General]
Has there been any progress in updating iPXE to use GCM or ECDHE based ciphers instead of the deprecated CBC ones it currently supports?

There has been some discussion in the past on this here:
https://forum.ipxe.org/showthread.php?tid=11985&highlight=ECDHE

My company is now faced with having to drop use of iPXE entirely because the use of CBC ciphers has come up as a finding during PCI vulnerability scans.  We can no longer use these old ciphers and maintain PCI compliance.  Surely others are in the same position.

**Comment by NiKiZe** [comment]
You could reach out for commercial support and that way fund the development of such features.

**Comment by stappersg** [comment]
Please provide  URL
